stdlib/public/core/VarArgs.swift

```diff
@@ -383,6 +383,25 @@ extension Double : _CVarArgPassedAsDouble, _CVarArgAligned {
   }
 }
 
+#if !os(Windows) && (arch(i386) || arch(x86_64))
+extension Float80 : CVarArg, _CVarArgAligned {
+  /// Transform `self` into a series of machine words that can be
+  /// appropriately interpreted by C varargs.
+  @inlinable // FIXME(sil-serialize-all)
```

Comment:
Do we need this `FIXME`?

Reply:
copypasta from the CVarArg conformances of Float, Double. All should be resolved together, but that's a separate PR.